src/lib/viewers/BaseViewer.js

```diff
@@ -370,10 +371,6 @@ class BaseViewer extends EventEmitter {
             if (event && event.scale) {
                 this.scale = event.scale;
             }
-
-            if (this.annotatorConf) {
```

Comment:
I would keep the initialization of annotations separate because there's no guarantee that when the annotations assets are loaded that the viewer would have loaded the content as well.

Reply:
try something like this
```
if (!this.annotationPromise) {
  return;
}

this.annotationPromise
    .then(this.onAnnotationsLoaded)
    .catch(() => { ... });
```

Reply:
done